store/sagas/user.js

```diff
@@ -0,0 +1,28 @@
+import {
+  put, takeLatest,
+} from 'redux-saga/effects';
+import {
+  LOGIN_REQUESTED,
+  LOGOUT_REQUESTED, loginSucceeded, logoutSucceeded,
+} from '../actions/user';
+
+
+function* requestLoginSaga() {
```

Comment:
You're gonna need to explain the generator stuff to me. Anything I can read up on it?